lib/fog/azurerm/models/dns/record_set.rb

```diff
@@ -50,6 +61,23 @@ def destroy
         def get_records(resource_group, name, zone_name, record_type)
           service.get_records_from_record_set(resource_group, name, zone_name, record_type)
         end
+
+        def update_ttl(ttl)
+          record_set = service.create_or_update_record_set(resource_group, name, zone_name, records, type.split('/').last, ttl)
```

Comment:
instead of using syntax like type.split('/'),
write and use utility function to get record type.